test/ffmpeg-vaapi/transcode/transcoder.py

```diff
@@ -10,35 +10,132 @@
 
 @slash.requires(have_ffmpeg)
 @slash.requires(have_ffmpeg_vaapi_accel)
+@platform_tags(ALL_PLATFORMS)
 class TranscoderTest(slash.Test):
+  requirements = dict(
+    decode = {
+      "avc" : dict(
+        sw = (ALL_PLATFORMS, (True, True), None),
```

Comment:
We need to specify the SW decoder here (similar to ffmpeg-qsv transcoder.py)

Reply:
added SW decoder validation check

Reply:
You check for it now, yes.  But we should also use the decoder explicitly, too.

Reply:
added the decoder